R/navs-legacy.R

```diff
@@ -203,29 +202,46 @@ navbar_options <- function(
   ...,
   position = c("static-top", "fixed-top", "fixed-bottom"),
   bg = NULL,
-  inverse = "auto",
+  type = c("auto", "light", "dark"),
   collapsible = TRUE,
   underline = TRUE
 ) {
   # Track user-provided arguments for print method and deprecation warnings
   is_default <- list(
     position = missing(position),
     bg = missing(bg),
-    inverse = missing(inverse),
+    type = missing(type),
```

Comment:
`type` was always a "least bad" choice, but after coming back to this I really don't like it. I'm going to replace `type` with `theme`, which seems to be a reasonable fit. `theme` matches how we end up using it i.e. `theme = "dark"` → `data-bs-theme="dark"`.

I also considered `style` (overlaps with HTML attribute) and `mode` (Bootstrap calls it "color mode", but mode isn't very clear or specific).